pom.xml

```diff
@@ -91,6 +91,7 @@ THE SOFTWARE.
     <project.patchManagement.url>https://api.github.com</project.patchManagement.url>
     <patch.tracker.serverId>jenkins-jira</patch.tracker.serverId>
 
+    <hamcrest.version>2.1</hamcrest.version>
     <matrix-auth.version>2.3</matrix-auth.version>
```

Comment:
Use dependencyManagement instead?

Reply:
jenkins-test-harness [doesn't do that| https://github.com/jenkinsci/jenkins-test-harness/blob/master/pom.xml#L110], why do this here differently?

Reply:
Less code, but no strong opinion